prow/cmd/deck/jobs.go

```diff
@@ -34,6 +34,9 @@ const (
 
 type Job struct {
 	Repo        string `json:"repo"`
+	Branch      string `json:"branch"`
+	HeadSHA     string `json:"head_sha"`
+	BaseSHA     string `json:"base_sha"`
```

Comment:
'HeadSHA' and 'BaseSHA' are confusing (HEAD has a specific meaning in Git vocabulary). Can you call it "BranchSHA"/"BranchRef" and "BaseRef" instead? (And comment what they mean)